region/lowernorfair/east/Main Hall.json

```diff
@@ -174,14 +174,50 @@
     {
       "id": 5,
       "link": [1, 4],
-      "name": "TrickyJumps",
+      "name": "Tricky Jumps",
       "requires": [
         "h_navigateHeatRooms",
         "canTrickyJump",
         {"heatFrames": 570}
       ],
       "note": "Take the platforms two at a time by building some run speed for each jump on a one tile runway."
     },
+    {
+      "link": [1, 4],
+      "name": "Tricky Jumps with Crystal Flash",
```

Comment:
You can cross with 2 Crystal Flashes no tanks.  Its not even that bad.
The 1 CF no tanks takes some frame maxing

Reply:
Ok, added strats for double and triple CF. For double CF, I could only get it with SpaceJump: https://videos.maprando.com/video/8214.

Reply:
https://videos.maprando.com/video/8222
I went for the extra jump on CF2 because I had slow elevators on, and that was unreasonable.  
I hit the elevator at 54 energy being more safe.  The first series of jumps has to be perfect though.